compiler-plugin/src/main/java/io/ballerina/stdlib/grpc/plugin/GrpcCompilerPluginConstants.java

```diff
@@ -43,7 +43,11 @@
                 "GRPC_104", DiagnosticSeverity.ERROR),
         TWO_PARAMS_WITHOUT_CALLER("when there are two parameters to a remote function, the first one " +
                 "must be a caller type", "GRPC_105", DiagnosticSeverity.ERROR),
-        INVALID_CALLER_TYPE("expected caller type \"", "GRPC_106", DiagnosticSeverity.ERROR);
+        INVALID_CALLER_TYPE("expected caller type \"", "GRPC_106", DiagnosticSeverity.ERROR),
+        EMPTY_SERVICE_NAME("invalid service name. Service name cannot be nil", "GRPC_107",
+                DiagnosticSeverity.ERROR),
+        HIERARCHICAL_SERVICE_NAME("invalid service name. Service name should not be a hierarchical name",
```

Comment:
Shall we add the service name to the error message?. It will be comprehensive. Something like:
```
invalid service name "/HelloWorld". Service name should not be a hierarchical name.
```

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/426/commits/587d96976ec29eeb8a51a98c58c97d659fff7e85